pam-u2f.c

```diff
@@ -250,8 +255,37 @@ int pam_sm_authenticate(pam_handle_t * pamh, int flags, int argc,
 
   if (cfg->manual == 0) {
     if (cfg->interactive) {
+      if (!cfg->prompt) {
+        buf = NULL;
+        prompt_len =
+            strlen(DEFAULT_PROMPT) + strlen("\n") + 1;
+        buf = malloc(sizeof(char) * (prompt_len));
+
+        if (!buf) {
+          DBG(("Unable to allocate memory"));
+          retval = PAM_IGNORE;
+          goto done;
+        }
+
+        strcpy(buf, DEFAULT_PROMPT);
+        strcat(buf, "\n");
+
+        DBG(("Using default prompt %s", buf));
+
+        cfg->prompt = strdup(buf);
```

Comment:
I setup it "copy&paste" same way as found for authfile parameter. 
But if it can done easier faster than feel free to improve it. I had several years not programmed C anymore.